core/include/Electron.h

```diff
@@ -22,6 +23,8 @@ class Electron : public Particle {
     throw std::runtime_error("unknown Electron::tag '" + tagname + "'");
   }
 
+  typedef class source_candidate source_candidate;
```

Comment:
this line was added only as a workaround for the previous 74X production in the RunII_25ns_v2 branch (see #274)
it was never needed in master; indeed the Muon class doesn't have it.
we should remove it. is it ok Thomas?